Sources/MapboxMaps/Annotations/AnnotationOrchestratorImpl.swift

```diff
@@ -53,14 +53,21 @@ internal final class AnnotationOrchestratorImpl: NSObject, AnnotationOrchestrato
         longPressGestureRecognizer.addTarget(self, action: #selector(handleDrag(_:)))
         longPressGestureRecognizer.delegate = self
         tapGestureRecognizer.delegate = self
+        longPressGestureRecognizer.isEnabled = false
+        tapGestureRecognizer.isEnabled = false
     }
 
     /// Dictionary of annotation managers keyed by their identifiers.
     internal var annotationManagersById: [String: AnnotationManager] {
         annotationManagersByIdInternal
     }
 
-    private var annotationManagersByIdInternal = [String: AnnotationManagerInternal]()
+    internal var annotationManagersByIdInternal = [String: AnnotationManagerInternal]() {
+        didSet {
+            longPressGestureRecognizer.isEnabled = !annotationManagersById.isEmpty
+            tapGestureRecognizer.isEnabled = !annotationManagersById.isEmpty
```

Comment:
> Missed this in the initial review - since this is the property observer of annotationManagersByIdInternal property it would be logical to use this property instead of the public wrapper here.

Would it be possible to address this?